### DISCUSSION TITLE
How to minimize Mesh Router?

### DISCUSSION THREAD

**Discussion by roborobs1023** [Ylianst/MeshCentral#6274] [Issues]
Unable to minimize MeshCentral Router. There is no minimize button and when you close it, it doesn't go to the system tray and keep running so it has to stay up on your screen. Any way around this?

![image](https://github.com/user-attachments/assets/ad186868-7d63-4c37-a553-af4e81b72a9a)

**Comment by si458** [comment]
what OS are you using as i see the minimise and max buttons here?
<img width="497" alt="image" src="https://github.com/user-attachments/assets/6a5133a9-a8b7-4b6e-9e1c-b2190f2bf132">

**Reply by si458** [reply]
have you also tried turning your computer off and on again?
and also making sure Windows isnt in anything daft like tablet mode? (which i believe hides buttons)

**Comment by DraconicMatty** [comment]
Click the "S" button on the bottom left of the mappings tab.

![image](https://github.com/user-attachments/assets/081f98f2-89ed-4ab0-82bf-bb4645a39a20)

You'll find the option to "show on system tray" there, which will keep it running in the system tray when you hit the "x" button in the top right. Note that this will not keep it open if you hit the "Close" button in the bottom right.
